File: src/main/java/no/nav/melosys/eessi/models/sed/nav/Aarsak.java
package no.nav.melosys.eessi.models.sed.nav;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Data;

@Data
@JsonIgnoreProperties(ignoreUnknown = true)
public class Aarsak {

    private String type;
    private String annet;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/models/sed/nav/Aarsak.kt
package no.nav.melosys.eessi.models.sed.nav

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
data class Aarsak(
    var type: String? = null,
    var annet: String? = null
)
